components/Extensions/samples/TextBoxExtensions.md

```diff
@@ -125,7 +125,7 @@ The `SurfaceDialOptions` property adds features from the Surface Dial control to
 
 ```xaml
```

Comment:
We didn't bring Surface Dial options over, did we?

Reply:
Oh, they appear to be here still: https://github.com/CommunityToolkit/Windows/tree/main/components/Extensions/src/Text

Not sure how they work with WASDK... something that probably needs to be tested...